consensus/hotstuff/eventloop/event_loop.go

```diff
@@ -30,19 +30,22 @@ type EventLoop struct {
 
 var _ hotstuff.EventLoop = (*EventLoop)(nil)
 var _ component.Component = (*EventLoop)(nil)
+var _ hotstuff.TimeoutCollectorConsumer = (*EventLoop)(nil)
 
 // NewEventLoop creates an instance of EventLoop.
 func NewEventLoop(log zerolog.Logger, metrics module.HotstuffMetrics, eventHandler hotstuff.EventHandler, startTime time.Time) (*EventLoop, error) {
 	proposals := make(chan *model.Proposal)
 	quorumCertificates := make(chan *flow.QuorumCertificate, 1)
+	timeoutCertificates := make(chan *flow.TimeoutCertificate, 1)
```

Comment:
It is likely that we have bursts of QCs (and maybe TCs) when the node is receiving timeouts or catching up. Per convention, consumers of notifications should not block. But I feel with a channel capacity of 1, it is likely to block for QCs. 


I see three options:
1. use channels with much higher capacity (which still leaves the problem of blocking consumer as a theoretical edge case)
2. use `fifoqueue` (which might drop messages, when they become full, which will undermine liveness in case of QCs and TCs)
3. only retain latest QC or TC (we can implement this with a spinning update similarly to `StrictMonotonousCounter` but with unsafe compare and swap pointers)

I think it is fine for this PR, but having an issue to make `EventLoop` consume notifications without blocking would be good.

Reply:
We have an issue for data processing queues for event loop: https://github.com/dapperlabs/flow-go/issues/6230
I will add this as comment as well.